frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -72,34 +71,45 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
     const displayOnly = !!display;
     const cloneBaseDiskImage = iGetFieldValue(cloneBaseDiskImageField);
     const mountWindowsGuestTools = iGetFieldValue(mountWindowsGuestToolsField);
+    const isUserTemplateValid = iGetIsLoaded(iUserTemplate) && !iGetLoadError(iUserTemplate);
+    const userTemplate = isUserTemplateValid ? iGetLoadedData(iUserTemplate).toJS() : null;
+    const userTemplateName: string = isUserTemplateValid
+      ? iGetName(iGetLoadedData(iUserTemplate))
+      : '';
     const params = {
-      userTemplate,
+      userTemplateName,
```

Comment:
can we remove it from params. It does not seem necessary to have it there.

Reply:
It is. we use it later on in this method and need it for `getFlavor` and `getWorkloadProfiles`